### ISSUE TITLE
error in cudnnBatchNormalizationForwardInference

### ISSUE THREAD

**Issue by seed93** [soumith/cudnn.torch#185]
Error occurred in cudnnBatchNormalizationForwardInference when input:size(1) > 1024.
cudnnBatchNormalizationForwardTraining is OK.

**Comment by ngimel** [comment]
That's expected, cudnn has this limitation. You can split your batch in two, for inference it is a pointwise calculation and results do not depend on batch size, or use nn.BatchNormalization

**Comment by pranerd** [comment]
The latest version of cudnn seems to have no such limitation,  try to update the cudnn.
